Makefile.package

```diff
@@ -151,9 +160,28 @@ info-channel:
 info-fqn:
 	echo ${PACKAGE_FQN}
 
+info: parse-info
+	echo ${PACKAGE_INFO}
+
 # CONFIGURATION TARGETS -------------------------------------------------------
 .PHONY: editable uneditable
 
+ifneq "${LOCKFILE_SOURCE}" ""
+LOCKFILE_OPTION := --lockfile="${BUILD_LOCKFILE}"
+${BUILD_LOCKFILE}: ${LOCKFILE_SOURCE}
+	# Create lockfile from LOCKFILE_SOURCE.
+	#
+	mkdir -p "${BUILD_DIR}"
+	conan lock create \
+		--lockfile-out "${BUILD_LOCKFILE}" \
+		--build=${BUILD_POLICY} \
+		-s ${PACKAGE_NAME}:build_type=${BUILD_TYPE} \
+		${CONAN_OPTIONS} \
+		"${LOCKFILE_SOURCE}"
+else
+.PHONY: ${BUILD_LOCKFILE}
```

Comment:
- [ ] Make always PHONY so that when `LOCKFILE_SOURCE` specified, a new lockfile is always created.

Reply:
This means though that something like:
```
make status LOCKFILE_SOURCE=../conanfile.py
```
will always build the lockfile anew.